specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/event-grid-schema/chat_thread_participant_removed.json

```diff
@@ -17,6 +17,10 @@
         "communicationUser": {
           "id": "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-5cbb-38e6-07fd-084822002467"
         }
+      },
+      "metadata": {
+        "key": "value",
+        "description": "A map of data associated with the message"
```

Comment:
Why is there a description entry in the example?

Reply:
This was the suggestion from Andrei Giurgiu to do it this way with the description. Is there another way you would prefer it? Should I remove that description value?

Reply:
The description goes in the swagger rather than the example. The example is meant to represent what a payload would look like so it wouldn't have descriptions of the properties.